Add Araxis Merge 2024.6001

gjmatthews opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/173254)

gjmatthews committed:
Add Araxis Merge 2024.6001

stephengillie commented:
Hi @gjmatthews,

This manifest adds a DisplayVersion to the AppsAndFeaturesEntries that isn't present in previous manifest versions. This entry should be added to the previous versions, or removed from this version.

(Automated response - build 895.)
<!--
[Policy] Needs-Author-Feedback
-->

gjmatthews commented:
> Hi @gjmatthews,
> 
> This manifest adds a DisplayVersion to the AppsAndFeaturesEntries that isn't present in previous manifest versions. This entry should be added to the previous versions, or removed from this version.
> 
> (Automated response - build 895.)

I have added the AppsAndFeaturesEntries to the previous manifest versions. Please see:

* https://github.com/microsoft/winget-pkgs/pull/173755
* https://github.com/microsoft/winget-pkgs/pull/173756

We have added the `AppsAndFeaturesEntries` entry to our manifests to hopefully fix a `winget upgrade` problem we are seeing. `winget upgrade` doesn't appear to be able to correctly match the installed version of Merge, such that repeated calls to `winget upgrade` continuously re-install the same version of Merge. This problem also occurs when testing a local copy of the updated manifest files: `winget upgrade --manifest <MANIFEST_DIR>`. Do our manifest files look correct with regards upgrade version checking?

gjmatthews commented:
@stephengillie Please kill this PR; see my comment on the related PR: https://github.com/microsoft/winget-pkgs/pull/173756#issuecomment-2356586001
